### PR TITLE
save active hall in session

### PR DESCRIPTION
**Pull request by BertPie**
FEATURE: save active hall in session
REFACTOR: move some logic out of create hall servlet

### CODE REVIEW

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/ActiveHallService.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import org.slf4j.LoggerFactory;
+
+import javax.ejb.Stateless;
+import javax.inject.Inject;
+import javax.servlet.http.HttpSession;
+
+
+@Stateless
+public class ActiveHallService {
+    private static final Logger LOG = LoggerFactory.getLogger(ChooseHallServlet.class);
```
Comment (MichalNowakowski) [inline ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/ActiveHallService.java]: nazwa loggera

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/ActiveHallService.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    private HallQueryService hallQueryService;
+
+    @Inject
+    private InputValidator inputValidator;
+
+
+    public Hall getActiveHall(HttpSession session){
+        Hall hall = (Hall) session.getAttribute("activeHall");
+
+        if(hall == null) {
```
Comment (MichalNowakowski) [inline ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/ActiveHallService.java]: formatowanie kodu

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/ActiveHallService.java (hunk 1 above)
Comment (BertPie) [reply ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/ActiveHallService.java]: Znowu... A już byłem pewien że się pilnuje...

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/ActiveHallService.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+    public boolean setActive(HttpSession session, String hid) {
+        Integer hallId = inputValidator.reqIntegerValidator(hid);
+        Hall hall = hallQueryService.findById(hallId);
+
+        if (hall == null) {
+            return false;
+        }
+
+        session.setAttribute("activeHall", hall);
```
Comment (isacademyci) [inline ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/ActiveHallService.java]: INFO Complete the task associated to this TODO comment. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1135)

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/ActiveHallService.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import org.slf4j.LoggerFactory;
+
+import javax.ejb.Stateless;
+import javax.inject.Inject;
+import javax.servlet.http.HttpSession;
+
+
+@Stateless
+public class ActiveHallService {
+    private static final Logger LOG = LoggerFactory.getLogger(ActiveHallService.class);
```
Comment (isacademyci) [inline ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/ActiveHallService.java]: MAJOR Remove this unused "LOG" private field. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)

**Comment by isacademyci** [comment]
SonarQube analysis reported 14 issues
* ![MAJOR][MAJOR] 6 major
* ![INFO][INFO] 8 info

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [HallQueryService.java#L14](https://github.com/infoshareacademy/jjdd5-niewiem/blob/ed04c8b49f87479af5c3d503f47d6f43c87b6981/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/HallQueryService.java#L14): Remove this unused "LOG" private field. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)
1. ![MAJOR][MAJOR] [TableSaveService.java#L16](https://github.com/infoshareacademy/jjdd5-niewiem/blob/ed04c8b49f87479af5c3d503f47d6f43c87b6981/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/TableSaveService.java#L16): Remove this unused "LOG" private field. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)
1. ![MAJOR][MAJOR] [DevPanelServlet.java#L24](https://github.com/infoshareacademy/jjdd5-niewiem/blob/ed04c8b49f87479af5c3d503f47d6f43c87b6981/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java#L24): Remove this unused "LOG" private field. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by isacademyci** [comment]
SonarQube analysis reported 13 issues
* ![MAJOR][MAJOR] 6 major
* ![INFO][INFO] 7 info

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [HallQueryService.java#L14](https://github.com/infoshareacademy/jjdd5-niewiem/blob/ed04c8b49f87479af5c3d503f47d6f43c87b6981/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/HallQueryService.java#L14): Remove this unused "LOG" private field. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)
1. ![MAJOR][MAJOR] [TableSaveService.java#L16](https://github.com/infoshareacademy/jjdd5-niewiem/blob/ed04c8b49f87479af5c3d503f47d6f43c87b6981/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/TableSaveService.java#L16): Remove this unused "LOG" private field. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)
1. ![MAJOR][MAJOR] [DevPanelServlet.java#L24](https://github.com/infoshareacademy/jjdd5-niewiem/blob/ed04c8b49f87479af5c3d503f47d6f43c87b6981/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java#L24): Remove this unused "LOG" private field. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by isacademyci** [comment]
SonarQube analysis reported 9 issues
* ![MAJOR][MAJOR] 3 major
* ![INFO][INFO] 6 info

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [HallQueryService.java#L14](https://github.com/infoshareacademy/jjdd5-niewiem/blob/680d6566a8d4b498932bffe756bbb6971166c33c/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/HallQueryService.java#L14): Remove this unused "LOG" private field. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)
1. ![INFO][INFO] [InputValidator.java#L14](https://github.com/infoshareacademy/jjdd5-niewiem/blob/680d6566a8d4b498932bffe756bbb6971166c33c/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/services/validators/InputValidator.java#L14): Complete the task associated to this TODO comment. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**